tools/pkg/repo/create.py

```diff
@@ -485,10 +485,12 @@ def rpm(
     if salt_repo_user and salt_repo_pass:
         repo_domain = f"{salt_repo_user}:{salt_repo_pass}@{repo_domain}"
 
-    def _create_repo_file(create_repo_path, url_suffix):
+    def _create_repo_file(create_repo_path, url_suffix, nightly_path=None):
         ctx.info(f"Creating '{repo_file_path.relative_to(repo_path)}' file ...")
         if nightly_build_from:
-            base_url = f"salt-dev/{nightly_build_from}/{datetime.utcnow().strftime('%Y-%m-%d')}/"
+            if not nightly_path:
```

Comment:
in hindsight, maybe this should be `if nightly_paths is none`

Reply:
yes, it should because passing `nightly_path` as `""` would make the above check `True` too. No sure why anyone would pass `""` but still...

Reply:
adjusted to `if nightly_path is None`